tests/tr1/tests/cstddef/test.cpp

```diff
@@ -20,16 +20,14 @@ typedef struct {
 } Str;
 
 void test_cpp() { // test C++ header
-    ptrdiff_t zero                  = 0;
-    static char* pc                 = (char*) zero;
     static const STDx size_t offs[] = {offsetof(Str, f1), offsetof(Str, f2), offsetof(Str, f3)};
-    STDx ptrdiff_t pdiff            = &pc[INT_MAX] - &pc[0];
     wchar_t wc                      = L'Z';
     Str x                           = {1, {2}, 3};
     char* ps                        = (char*) &x;
+    STDx ptrdiff_t pdiff            = &ps[INT_MAX] - &ps[0];

```

Comment:
Oh, `INT_MAX` is too large to be valid. The value should not be large than the length of the array (or 1 for pointers to non-element objects).